provisioning/device/src/tpm_registration.ts

```diff
@@ -122,7 +125,8 @@ export class TpmRegistration extends EventEmitter implements RegistrationClient
             - `sasToken`: the SAS token generated according to `SRS_NODE_DPS_TPM_REGISTRATION_16_006`
             - `registrationInfo`: an object with the following properties `endorsementKey`, `storageRootKey`, `registrationId` and their previously set values.
             - a callback that will handle an optional error and a `result` object containing the IoT hub name, device id and symmetric key for this device.]*/
-            this._transport.register(registrationInfo, sasToken, (err, result) => {
+            this._transport.setSasToken(sasToken);
```

Comment:
>setSasToken [](start = 28, length = 11)

oh I see. well. since we have a getAuthenticationChallenge I'd suggest we name this "sendAuthenticationChallengeResponse" or something along those lines. Even though it looks like a SAS token i'm not sure that it's one (tokens are usually associated with CBS)